graylog2-server/src/main/java/org/graylog2/initializers/IndexerSetupService.java

```diff
@@ -61,8 +61,8 @@
 @Singleton
 public class IndexerSetupService extends AbstractIdleService {
     private static final Logger LOG = LoggerFactory.getLogger(IndexerSetupService.class);
-    private static final Version MINIMUM_ES_VERSION = Version.V_1_3_4;
-    private static final Version MAXIMUM_ES_VERSION = Version.fromString("1.7.99");
+    private static final Version MINIMUM_ES_VERSION = Version.V_2_0_0;
+    private static final Version MAXIMUM_ES_VERSION = Version.fromString("2.0.99");
```

Comment:
Although we could discuss whether this check (for the maximum version) is still required or if it should simply be set to "2.99.99".